index.php

```diff
@@ -39,25 +43,78 @@ function enqueue_scripts($pages) {
 
     wp_register_script( 'ace', 'https://cdn.jsdelivr.net/ace/1.2.3/min/ace.js' );
     wp_enqueue_script( 'ace' );
-
 }
 
 function init_options_page(){
+    main();
+}
+
+function main() {
+    global $wpdb;
+    $plugin = new \QA\Plugin($wpdb, $wpdb->prefix . 'qaUsers6', 4);
```

Comment:
Apparently you forgot to restore a dynamic user ID in this call (third parameter). Also, the name of the table looks a bit weird (why the 6?).